src/modules/tests/test-data/cat-adi-part2/vehicle-checks/vehicle-checks.cat-adi-part2.reducer.ts

```diff
@@ -4,26 +4,60 @@ import {
   NUMBER_OF_TELL_ME_QUESTIONS as numberOfTellMeQuestions,
 }
 from '../../../../../shared/constants/tell-me-questions/tell-me-questions.cat-adi-part2.constants';
-import {
-  NUMBER_OF_SHOW_ME_QUESTIONS as numberOfShowMeQuestions,
-}
-from '../../../../../shared/constants/show-me-questions/show-me-questions.cat-adi-part2.constants';
+import { CompetencyOutcome } from '../../../../../shared/models/competency-outcome';
 
 export const initialState: CatADI2UniqueTypes.VehicleChecks = {
   tellMeQuestions: Array(numberOfTellMeQuestions).fill({}),
-  showMeQuestions: Array(numberOfShowMeQuestions).fill({}),
+  showMeQuestions: [
```

Comment:
maybe create a const 
const showMeQuestion: QuestionResult = { code: '', description: '', outcome: CompetencyOutcome.P };
then have
Array(numberOfShowMeQuestions).fill(showMeQuestion);

leave it up to you if you decide to or not